testapi.pm

```diff
@@ -2326,8 +2216,7 @@ A typical call would look like:
 
 =cut
 
-sub compat_args {
-    my ($def_args, $fix_keys) = splice(@_, 0, 2);
+sub compat_args ($def_args, $fix_keys, @) {
     my %ret;
     for my $key (@{$fix_keys}) {
         $ret{$key} = shift if (scalar(@_) >= 1 && (!defined($_[0]) || !grep { $_ eq $_[0] } keys(%{$def_args})));
```

Comment:
`@_`

Reply:
`# no:style:signatures` is fine by me I guess